Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -202,6 +202,19 @@ public extension Sequence {
     func sorted<T: Comparable>(by keyPath: KeyPath<Element, T>) -> [Element] {
         return sorted { $0[keyPath: keyPath] < $1[keyPath: keyPath] }
     }
+
+    /// SwifterSwift: Sum of a `Numeric` property of each `Element` in a `Sequence`.
+    ///
+    ///     ["James", "Wade", "Bryant"].sum(for: \.count) -> 15
+    ///
+    /// - Parameter keyPath: Key path of the `Numeric` property.
+    /// - Returns: The sum of the `Numeric` propertys at `keyPath`.
+    func sum<T: Numeric>(for keyPath: KeyPath<Element, T>) -> T {
```

Comment:
If you're already going for a generic implementation, why not go up a step to `AdditiveArithmetic`?
```suggestion
    func sum<T>(for keyPath: KeyPath<Element, T>) -> T where T: AdditiveArithmetic {
```

Reply:
I think a type constraint is enough here, no need for a generic where clause. What do you think?

Reply:
They're actually the exact same, I just prefer always using `where` (up to you, personally). You can do it whichever way you like 😄

Reply:
Okay, got it. All the other generic functions in the Sequence extension use the type constraint syntax, so I'd like to keep it for this one as well 😇